### PR TITLE
bpo-37163: Make the obj argument of dataclasses.replace() a positional-only.

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
https://bugs.python.org/issue37163

### CODE REVIEW

**Review by vstinner — APPROVED** [review]
LGTM.

File: Doc/library/dataclasses.rst [hunk 1]
```diff
@@ -356,7 +356,7 @@ Module-level decorators, classes, and functions
          def add_one(self):
              return self.x + 1
 
-.. function:: replace(instance, **changes)
+.. function:: replace(instance, /, **changes)
```
Comment (vstinner) [inline Doc/library/dataclasses.rst:359]: Maybe add a versionchanged 3.9 markup to document this subtle change?

Comment (serhiy-storchaka) [reply Doc/library/dataclasses.rst:359]: It was not added in other 35 cases. If it is needed it should be added in all cases.

**Comment by vstinner** [comment]
I am also fine with not documenting the change. That's why I approved your PR 😉

**Pull request merged**